frontend/src/pages/Orientation/Pages/RelativeLocation.tsx

```diff
@@ -124,48 +125,43 @@ function RelativeLocation(props: any) {
 		return postal;
 	}
 
-	return (
-		<div className="location-page">
-			<h3 className="main-heading">Discover your community.</h3>
-			<p className="sub-heading">
-				Enter your postal code to have your account better suit your
-				needs.
-			</p>
-			<div className="location-container">
-				<InputMask
-					mask="a9a 9a9"
-					maskChar={null}
-					value={postalCode}
-					className="input"
-					onChange={handlePostalCodeChange}
-					alt="Postal Code"
-					title="Postal Code Input"
-					placeholder="Postal Code"
-					type="text"
-				/>
-				<img src="/icons/location.png" className="location-icon" />
-			</div>
-			<div className="map">
-				{isLoaded &&
-					postalCode.length === POSTAL_CODE_LEN &&
-					!inputError && (
-						<GoogleMap
-							mapContainerStyle={mapContainerStyle}
-							center={center}
-							zoom={MAP_ZOOM_LEVEL}
-						>
-							{/* Dynamic map content */}
-						</GoogleMap>
-					)}
-			</div>
-			{postalCode.length === POSTAL_CODE_LEN && inputError && (
-				<p className="error">Please insert a valid postal code.</p>
-			)}
-			<button onClick={getBrowserLocation}>
-				Get&nbsp;Current&nbsp;Location
-			</button>
-		</div>
-	);
+    return (
+        <div className="location-page">
+            <h3 className="main-heading">Discover your community.</h3>
+            <p className="sub-heading">Enter your postal code to have your account better suit your needs.</p>
+                <div className="location-container">
+                {/* Note: The new input mask causes issues when trying to put a space in between the postal code, 
+                the fix in handlePostalCodeChange which was created for the old library does not work anymore. */}
+                    <InputMask
+                        mask="_!_!_!"
```

Comment:
```suggestion
                        mask="_!_ !_!"
```

In addition, try 

- changing the `POSTAL_CODE_LEN` constant to 7, and
- removing the `replace` call on the returned postal code in the `convertToPostal()` routine.

From a quick glance, these changes seem to address the space issue on my end. Let me know what happens on your end.

Reply:
Tested it out on my end and it looks like you fix solved the issues. Changes have been pushed on commit [326b51f](https://github.com/projecthuman-repo/Coquest-WebApp/pull/73/commits/326b51f11aa2c69650d89e5a76fdeee1df61cfc9). Let me know how it looks.